activerecord/test/cases/autosave_association_test.rb

```diff
@@ -1995,6 +1995,7 @@ class TestCyclicAutosaveAssociationsOnlySaveOnce < ActiveRecord::TestCase
   test "saving? is reset to false if validations fail" do
     child = Ship.new(name: "Nights Dirty Lightning")
     child.build_pirate
+    def child.saving?; @saving; end
```

Comment:
Do we need to define this method? I'd just check the result of it being fail or the instance variable directly.

Reply:
Indeed, I've removed `saving?` entirely.